### ISSUE TITLE
Filter by startsWith instead of contains the typed value

### ISSUE THREAD

**Issue by pablovegau** [based-ghost/react-functional-select#4]
Is it possible to add a prop to filter by startsWith instead of the current filter?

If I have these list of words: 

- strike
- ancestor
- indication
- recognize
- dilute
- man
- fog
- rise
- grudge
- recognize

Currently, if I search `ri` the result is:

- st**ri**ke
- **ri**se

If we apply the new proposed filter the result is: 

- **ri**se

Thanks

**Comment by based-ghost** [comment]
@pablovegau I have added the requested functionality in the latest release (v1.2.1). After updating the package, you will then have a new property `filterMatchFrom` that can be set to 'start' or 'any' (default value is 'any'). So in your case you can hard code a value of 'start' or use the exported `FilterMatchEnum`:

```js
import { Select, FilterMatchEnum } from 'react-functional-select';

<Select
   options={options}
   filterMatchFrom={FilterMatchEnum.START}
/>
```

**Comment by based-ghost** [comment]
Closing issue - resolved in commit 826f653.
